docs/neural_modeling/plot_06_calcium_imaging.py

```diff
@@ -103,26 +126,26 @@
 coupling_basis = nmo.basis.RaisedCosineBasisLog(3, mode="conv", window_size=10)
 
 # %%
-# We need to make sure the design matrix will be full-rank by applying identifiability constraints
-# to the Cyclic Bspline
-heading_basis.identifiability_constraints = True
+# We need to make sure the design matrix will be full-rank by applying identifiability constraints to the Cyclic Bspline, and then combine the bases (the resturned object will be an `AdditiveBasis` object).
 
-# Here we combine the basis. The returned object is an `AdditiveBasis` object.
+heading_basis.identifiability_constraints = True
 basis = heading_basis + coupling_basis
 
+
 # %%
 # ## Gamma GLM
 #
-# Since the transients are non-negative, we will use a Gamma distribution from `nemos` with a soft-plus non linearity.
-#
+# Until now, we have been modeling spike trains, and have used a Poisson distribution for the observability model. With calcium traces, things are quite different: we no longer have counts but continuous signals, so the Poisson is no longer appropriate. We cannot use a Gaussian because the calcium traces are non-negative. To satisfy these constraints, we will use a Gamma distribution from `nemos` with a soft-plus non linearity.
+
 
 model = nmo.glm.GLM(
     regularizer=nmo.regularizer.UnRegularized(solver_name="LBFGS", solver_kwargs=dict(tol=10**-13)),
     observation_model=nmo.observation_models.GammaObservations(inverse_link_function=jax.nn.softplus)
 )
 
 # %%
-# We select a neuron to fit and remove it from the list of predictors
+# We select one neuron to fit later, so remove it from the list of predictors
```

Comment:
In this case, we are removing the self-coupling from the predictors. If this wasn't clear to you. @EricThomson we should rephrase it. The neuron was left out because of the long auto-corr time; the autocorr filter resulted in over fitting.